consensus/src/commons.rs

```diff
@@ -284,30 +284,30 @@ impl AgreementSender {
 
     /// Sends an agreement (internally) to the agreement loop.
     pub(crate) async fn send(&self, msg: Message) -> bool {
-        if let Payload::Agreement(agreement) = &msg.payload {
-            if agreement.signature == [0u8; 48]
-                || agreement.first_step.is_empty()
-                || agreement.second_step.is_empty()
+        if let Payload::Quorum(q) = &msg.payload {
+            if q.signature == [0u8; 48]
+                || q.validation.is_empty()
+                || q.ratification.is_empty()
                 || msg.header.block_hash == [0; 32]
             {
                 return false;
             }
 
             tracing::debug!(
-                event = "send agreement",
+                event = "send quorum_msg",
                 hash = to_str(&msg.header.block_hash),
                 round = msg.header.round,
                 step = msg.header.step,
-                first = format!("{:#?}", agreement.first_step),
-                second = format!("{:#?}", agreement.second_step),
-                signature = to_str(&agreement.signature),
+                first = format!("{:#?}", q.validation),
```

Comment:
I'd rename `first` and `second` to be coherent

Reply:
Done